test/setup-database.js

```diff
@@ -52,6 +53,46 @@ exports.seedDatabase = database => () => {
     });
 };
 
+exports.seedICVDatabase = database => () => {
```

Comment:
Do we need this? Can't we just also load this file with the normal database seeding? So every database has everything?